src/openssl/ssl/server.cr

```diff
@@ -60,18 +63,24 @@ class OpenSSL::SSL::Server
   #
   # This method calls `@wrapped.accept` and wraps the resulting IO in a SSL socket (`OpenSSL::SSL::Socket::Server`) with `context` configuration.
   def accept : OpenSSL::SSL::Socket::Server
-    OpenSSL::SSL::Socket::Server.new(@wrapped.accept, @context, sync_close: @sync_close)
+    new_server(@wrapped.accept)
   end
 
   # Implements `::Socket::Server#accept?`.
   #
   # This method calls `@wrapped.accept?` and wraps the resulting IO in a SSL socket (`OpenSSL::SSL::Socket::Server`) with `context` configuration.
   def accept? : OpenSSL::SSL::Socket::Server?
     if socket = @wrapped.accept?
-      OpenSSL::SSL::Socket::Server.new(socket, @context, sync_close: @sync_close)
+      new_server(socket)
     end
   end
 
+  private def new_server(io)
```

Comment:
I'd rather call this `new_server_socket` or `new_ssl_socket`. It doesn't create a server (`self` is already the server).

Reply:
Done